src/frontend/utils/conversejs/converse-plugins/participantsTrackingPlugin.ts

```diff
@@ -12,36 +12,26 @@ const addParticipantsTrackingPlugin = () =>
       _converse.on('initialized', () => {
         _converse.connection.addHandler(
           (stanza: HTMLElement) => {
-            if (
-              stanza.getAttribute('from') !== null &&
-              stanza.getAttribute('to') !== null
-            ) {
-              const participantDisplayName = getNameFromJID(
-                stanza.getAttribute('from')!,
-              );
+            const jid = stanza.getAttribute('from');
```

Comment:
we will really have to do something because this file is quite a mess

Reply:
opening of an issue about it : #1466